packages/app-store/dailyvideo/lib/VideoApiAdapter.ts

```diff
@@ -118,7 +118,7 @@ const DailyVideoApiAdapter = (): VideoApiAdapter => {
           enable_screenshare: true,
           enable_chat: true,
           exp: exp,
-          enable_recording: "local",
+          enable_recording: "cloud",
```

Comment:
@Udit-takkar Just checking this decision, with this every cal video would do cloud recording and all users should be able to record but they won't be able to view it till they belong to a team plan.

Reply:
yeah they would see upgrade button in the ui https://www.figma.com/file/xk4HOxtSI82J0F7enMxeak/Cal---Live?node-id=933%3A108707&t=cJHMTav0G755y2B0-0